coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,167 @@
+        '.jl': {'Julia'},
+        '.lua': {'Lua'},
+        '.nse': {'Lua'},
+        '.rbxs': {'Lua'},
+        '.markdown': {'Markdown'},
+        '.md': {'Markdown'},
+        '.mkd': {'Markdown'},
+        '.mkdown': {'Markdown'},
+        '.ron': {'Markdown'},
+        '.matlab': {'Matlab', 'Octave'},
+        '.m': {'Objective-C', 'Matlab'},
+        '.mm': {'Objective-C'},
+        '.cl': {'OpenCL'},
+        '.opencl': {'OpenCL'},
+        '.aw': {'PHP'},
+        '.ctp': {'PHP'},
+        '.php': {'PHP'},
+        '.php3': {'PHP'},
+        '.php4': {'PHP'},
+        '.php5': {'PHP'},
+        '.phpt': {'PHP'},
+        '.nqp': {'Perl'},
+        '.perl': {'Perl'},
+        '.ph': {'Perl'},
+        '.pl': {'Perl'},
+        '.plx': {'Perl'},
+        'pm': {'Perl'},
+        '.pm6': {'Perl'},
+        '.pod': {'Perl'},
+        '.psgi': {'Perl'},
+        '.gyp': {'Python'},
+        '.py': {'Python'},
+        '.pyt': {'Python'},
+        '.pyw': {'Python'},
+        '.wsgi': {'Python'},
+        '.xpy': {'Python'},
+        '.r': {'R'},
+        '.builder': {'Ruby'},
+        '.gemspec': {'Ruby'},
+        '.god': {'Ruby'},
+        '.irbrc': {'Ruby'},
+        '.mspec': {'Ruby'},
+        '.podspec': {'Ruby'},
+        '.rb': {'Ruby'},
+        '.rbuild': {'Ruby'},
+        '.rbw': {'Ruby'},
+        '.rbx': {'Ruby'},
+        '.ru': {'Ruby'},
+        '.thor': {'Ruby'},
+        '.watchr': {'Ruby'},
+        '.scss': {'SCSS'},
+        '.sql': {'SQL'},
+        '.scl': {'Scala'},
+        '.scala': {'Scala'},
+        '.aux': {'TeX'},
+        '.bib': {'TeX'},
+        '.dtx': {'TeX'},
+        '.ins': {'TeX'},
+        '.ltx': {'TeX'},
+        '.mkii': {'TeX'},
+        '.mkiv': {'TeX'},
+        '.mkvi': {'TeX'},
+        '.sty': {'TeX'},
+        '.tex': {'TeX'},
+        '.toc': {'TeX'},
+        '.ts': {'TypeScript'},
+        '.vhdl': {'VHDL'},
+        '.v': {'Verilog'},
+        '.veo': {'Verilog'},
+        '.axml': {'XML'},
+        '.ccxml': {'XML'},
+        '.clixml': {'XML'},
+        '.cproject': {'XML'},
+        '.dita': {'XML'},
+        '.ditamap': {'XML'},
+        '.ditaval': {'XML'},
+        '.glade': {'XML'},
+        '.grxml': {'XML'},
+        '.jelly': {'XML'},
+        '.kml': {'XML'},
+        '.mxml': {'XML'},
+        '.plist': {'XML'},
+        '.pluginspec': {'XML'},
+        '.ps1xml': {'XML'},
+        '.psc1': {'XML'},
+        '.pt': {'XML'},
+        '.rdf': {'XML'},
+        '.rss': {'XML'},
+        '.scxml': {'XML'},
+        '.svg': {'XML'},
+        '.tmcommand': {'XML'},
+        '.tml': {'XML'},
+        '.tmlanguage': {'XML'},
+        '.tmpreferences': {'XML'},
+        '.tmsnippet': {'XML'},
+        '.tmtheme': {'XML'},
+        '.ui': {'XML'},
+        '.vxml': {'XML'},
+        '.wsdl': {'XML'},
+        '.wxi': {'XML'},
+        '.wxl': {'XML'},
+        '.wxs': {'XML'},
+        '.x3d': {'XML'},
+        '.xaml': {'XML'},
+        '.xlf': {'XML'},
+        '.xliff': {'XML'},
```

Comment:
hm, all those xml stuff are actually own languages based on XML, not sure we want to list xml here